lib/main.coffee

```diff
@@ -108,6 +108,7 @@ module.exports =
         parameters = @parameters.filter (item) -> item
         command = @command
         confFile = helpers.find(path.dirname(filePath), ['phpcs.xml', 'phpcs.xml.dist', 'phpcs.ruleset.xml'])
+        cwd = path.dirname(confFile) if confFile
```

Comment:
Won't this be undefined if `confFile` isn't "truthy"? It should default to the directory of the file if a `confFile` isn't found.